bin/prolfquapp

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This script is used to run a single command in the prolfquapp docker image.
+# - The docker image is pulled if it does not exist locally.
+# - The current directory will be mounted to /work and set as the current working directory for the command being executed.
+# Limitations
+# - Currently, specified paths should be relative to the current working directory and not go outside of it.
+
+set -euo pipefail
+
+# Default values for arguments
+IMAGE_VERSION="0.0.1"
+IMAGE_REPO="docker.io/leoschwarz/prolfquapp"
```

Comment:
Currently, this is going through my Docker Hub account but I would suggest to create an organization either for prolfqua or FGCZ.

Currently deployment is manual, we can set up a GitHub actions pipeline again later but it needs to be slightly adjusted for Docker Hub.